backend/tarsy/config/settings.py

```
@@ -155,25 +130,110 @@
     def get_llm_config(self, provider: str) -> Dict:
         """Get LLM configuration for a specific provider."""
         if provider not in self.llm_providers:
-            raise ValueError(f"Unsupported LLM provider: {provider}")
+            available = list(self.llm_providers.keys())
+            raise ValueError(f"Unsupported LLM provider: {provider}. Available: {available}")
```

Ruff (0.12.2)

Line too long (93 > 88)

(E501)